RawVideo/VideoFrameRenderer.cs

```diff
@@ -57,17 +53,19 @@ public unsafe void RenderRawVideoFrame(RawVideoFrameBuffer videoFrameBuffer)
                 destArrayBuffer[i + 3] = sourceArrayBuffer[i + 3];
             }
 
-            imageElement.Source = writeableBitmap;
+            imageElement.Source = bitmap;
+            (imageElement.Parent as RelativePanel).Background = new SolidColorBrush(Colors.Black);
```

Comment:
I am a bit bothered by this line :).  `as` does not guarantee a successful casting, and will return null if failed to cast. This line basically proceeds to access Background regardless. How about add a ? before .?

Reply:
i will do it even thought i know it will succeed because the parent is a relative panel

Reply:
Yes, let's do a plain if null check here, I think it's better to adhere a more common/standard style as a sample code shipped from MS.

Reply:
actually i moved that line to mainpage to avoid the cast every time there is a new frame